README.md

```diff
@@ -30,6 +30,11 @@ See https://github.com/hashicorp/vault/blob/main/api/client.go for the available
 In addition, the following environment variables can be set:
 
 - `VAULT_PATHPREFIX`: the path prefix to use for the Vault keys, which generally matches the secret store name (defaults to `kv`).
+- `VAULT_PATHNAME`: the path name to use for the Vault keys, which generally matches the secret store name (defaults to `nuts-private-keys`).
```

Comment:
Weird default for hashicorp-vault-proxy, I think we can live with 1 thing to configure (other than Vault adress/auth)?

Reply:
The default is to support the current vault implementation. So if you don't config anything, it is a drop-in replacement for the current vault store.